tests/testthat/test-ChoicesFilterState.R

```diff
@@ -1,64 +1,71 @@
 testthat::test_that("The constructor accepts character or factor", {
-  testthat::expect_no_error(ChoicesFilterState$new("test", varname = "test"))
-  testthat::expect_no_error(ChoicesFilterState$new(as.factor("test"), varname = "test"))
+  testthat::expect_no_error(ChoicesFilterState$new("test", x_reactive = reactive(NULL), varname = "test"))
+  testthat::expect_no_error(ChoicesFilterState$new(as.factor("test"), reactive(NULL), varname = "test"))
 })
 
+testthat::test_that("The constructor reactive as x_reactive", {
```

Comment:
`testthat::test_that("The constructor accepts only reactive input for x_reactive", {`